pkg/api/plugins.go

```diff
@@ -562,3 +562,33 @@ func mdFilepath(mdFilename string) (string, error) {
 		return "", ErrUnexpectedFileExtension
 	}
 }
+
+func (hs *HTTPServer) CheckEnabled(c *contextmodel.ReqContext) {
+	pluginID := web.Params(c.Req)[":pluginId"]
+	p, exists := hs.pluginStore.Plugin(c.Req.Context(), pluginID)
+	if !exists {
+		c.JsonApiErr(404, "Plugin not found", nil)
```

Comment:
Nit: For (desired) consistency, would be nice to use the `http` package for the status codes